.github/ISSUE_TEMPLATE/azure_functions_issue.md

```diff
@@ -0,0 +1,6 @@
+---
+name: Issue with Azure Functions
+about: Need help with Azure Functions
+---
+
+If you're running the application in Azure Functions, get support [here](TBD).
```

Comment:
https://github.com/Azure/azure-webjobs-sdk
https://github.com/Azure/azure-functions-host
We can start with these.